en_US/mqtt/auto_subscribe/auto_subscribe.md

```diff
@@ -0,0 +1,93 @@
+# MQTT
+
+## Auto Subscribe
+
+When a device is successfully connected to EMQX, no additional subscription requests (SUBSCRIBE messages) need to be sent and EMQX completes the pre-defined subscription relationship for the device. This feature is improved from the MQTT proxy subscription feature in EMQX version 4.
```

Comment:
We need more user story here.
Maybe give an example:

Reply:
Some examples in `Quick Start` part. I'm wondering if I should put the `Quick Start` part in the first chapter